src/main/java/org/apache/flink/benchmark/SinkUpsertMaterializerBenchmark.java

```diff
@@ -0,0 +1,298 @@
+SinkUpsertMaterializerBenchmark.run           false            100             200                  100         ROCKSDB            V1  thrpt         30.935          ops/ms
+SinkUpsertMaterializerBenchmark.run           false            100            1000                  100            HEAP            V1  thrpt        546.826          ops/ms
+SinkUpsertMaterializerBenchmark.run           false            100            1000                  100         ROCKSDB            V1  thrpt          7.081          ops/ms
+SinkUpsertMaterializerBenchmark.run            true            100               1                  100            HEAP            V1  thrpt       4006.263          ops/ms
+SinkUpsertMaterializerBenchmark.run            true            100               1                  100         ROCKSDB            V1  thrpt        297.556          ops/ms
+SinkUpsertMaterializerBenchmark.run            true            100              10                  100            HEAP            V1  thrpt       3240.089          ops/ms
+SinkUpsertMaterializerBenchmark.run            true            100              10                  100         ROCKSDB            V1  thrpt        209.375          ops/ms
+SinkUpsertMaterializerBenchmark.run            true            100             100                  100            HEAP            V1  thrpt       2131.445          ops/ms
+SinkUpsertMaterializerBenchmark.run            true            100             100                  100         ROCKSDB            V1  thrpt         78.209          ops/ms
+SinkUpsertMaterializerBenchmark.run            true            100             200                  100            HEAP            V1  thrpt        652.936          ops/ms
+SinkUpsertMaterializerBenchmark.run            true            100             200                  100         ROCKSDB            V1  thrpt         29.674          ops/ms
+SinkUpsertMaterializerBenchmark.run            true            100            1000                  100            HEAP            V1  thrpt        118.567          ops/ms
+SinkUpsertMaterializerBenchmark.run            true            100            1000                  100         ROCKSDB            V1  thrpt          6.426          ops/ms
+ * }</pre>
+ */
+@OperationsPerInvocation(value = SinkUpsertMaterializerBenchmark.RECORDS_PER_INVOCATION)
+@SuppressWarnings("ConstantValue")
+public class SinkUpsertMaterializerBenchmark extends BenchmarkBase {
+
+    private static final int STREAM_KEY = 0;
+
+    public static void main(String[] args) throws RunnerException {
+        new Runner(new OptionsBuilder()
+                .verbosity(VerboseMode.NORMAL)
+                // speedup
+//                .warmupIterations(1)
+//                .measurementIterations(1)
+//                .forks(1)
+//                .warmupTime(TimeValue.milliseconds(100))
+//                .measurementTime(TimeValue.seconds(1))
```

Comment:
left overs?

Reply:
I wanted to keep it to quickly test changes locally, but I can remove

Reply:
Please remove. You can also easily change that in the `BenchmarkBase`. (I almost aloways have have locally unocommited changes to `BenchmarkBase.java`)